pkg/tfgen/generate.go

```diff
@@ -856,6 +864,7 @@ func NewGenerator(opts GeneratorOptions) (*Generator, error) {
 		skipExamples:     opts.SkipExamples,
 		coverageTracker:  opts.CoverageTracker,
 		editRules:        getEditRules(info.DocRules),
+		noDocsRepo:       opts.XInMemoryDocs,
```

Comment:
why not name it xInMemoryDocs for symmetry?

Reply:
Because we also set `noDocsRepo` when we fail to find a repo during normal docs generation.